### PR TITLE
fuse-overlayfs: fixed paths in configure

### PR DESCRIPTION
**Pull request by henryhchchc**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fuse-overlayfs.rb [hunk 1]
```diff
@@ -18,9 +18,9 @@ class FuseOverlayfs < Formula
 
   def install
     system "autoreconf", "-fis"
-    system "./configure", "--prefix=/", "--sbindir=/sbin"
+    system "./configure", "--prefix=#{prefix}", "--sbindir=#{sbin}"
```
Comment (gromgit) [inline Formula/fuse-overlayfs.rb:21]:
```suggestion
    system "./configure", "--prefix=#{prefix}"
```
Nothing is installed in `../sbin`, and the default `configure` value is already correct anyway:
```bash
$ ./configure --help
[...]
  --prefix=PREFIX         install architecture-independent files in PREFIX
                          [/usr/local]
  --exec-prefix=EPREFIX   install architecture-dependent files in EPREFIX
                          [PREFIX]
[...]
  --sbindir=DIR           system admin executables [EPREFIX/sbin]
```

**Comment by gromgit** [comment]
Also, please do the following:
1. Add `revision 1` before the `license` line, so that everyone who's already installed this formula will get the updated bottle on a `brew upgrade`.
2. Fix the misspelled name in your Git commit (`fuse-overlays -> fuse-overlayfs`), otherwise this fix will never be found during a `brew extract` or any other Homebrew operation that searches Git logs based on commit names.

**Comment by henryhchchc** [comment]
Done. And I squashed the commits BTW.

**Review by carlocab — APPROVED** [review]
Thanks!

FYI, we leave out the version number for PRs that don't update the version. Cleaned that up for you.

**Pull request closed without merging**